OWNERS

```diff
@@ -2,7 +2,4 @@
 #  https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md
 
 approvers:
-  - hjacobs
-  - raffo
-  - linki
-  - njuettner
+  - seanmalloy
```

Comment:
I'm not a member of the `openshift` GitHub org, so I'm not the correct person to add here. Which people in the `openshift` GitHub org should be added as approvers and reviewers?

Reply:
@chris-short let me know if you need any additional info. Thanks!

Reply:
@thoraxe was able to get me in contact with @mcurry-rh.